src/squid.md.in

```diff
@@ -0,0 +1,234 @@
+: Do not catch fatal signals.
+
+-d level
+: Write debugging to stderr also.
+
+-f file
+: Use the given config-file instead of @SYSCONFDIR@/squid.conf .
+  If the file name starts with a **!** or **|** then it is
+  assumed to be an external command or command line. Can for
+  example be used to pre\-process the configuration before it is
+  being read by Squid. To facilitate this Squid also understands
+  the common #line notion to indicate the real source file.
+
+-F
+: Don't serve any requests until store is rebuilt.
+
+-h
+: Print help message.
+
+-i
+: Install as a Windows Service (see **"-n"** option).
+
+-k reconfigure | rotate | shutdown | interrupt | kill | debug |
+ check | parse
+: Parse configuration file, then send signal to running copy
+  (except **"-k parse"**) and exit.
+
+-l facility
+: Use specified syslog facility. Implies **"-s"**
+
+-n name
+: Specify Windows Service name to use for service operations.
+  The default is **"Squid"**
+
+-N
+: No daemon mode.
+
+-\-foreground
+: Parent process does not exit until its children have finished.
+  It has no effect with **"-N"** which does not fork/exit at
+  startup.
+
+-\-kid roleID
+: Play a given SMP kid process role, with a given ID.
+  **Do not use this option**. It is meant for the master process
+  use only.
+
+-O options
+: Set Windows Service Command line options in Registry.
+
+-r
+: Remove a Windows Service (see **"-n"** option).
+
+-R
+: Do not set **REUSEADDR** on port.
+
+-s
+: Enable logging to syslog. Also configurable in
+  **@SYSCONFDIR@/squid.conf**
+
+-S
+: Double-check swap during rebuild.
+
+-u port
+: Specify ICP port number.
+
+-v
+: Print version and build details.
+
+-X
+: Force full debugging.
+
+-Y
+: Only return **UDP_HIT** or **UDP_MISS_NOFETCH** during fast
+  reload.
+
+-z
+: Create missing swap directories and other missing
+  [cache_dir](http://www.squid-cache.org/Doc/config/cache_dir/)
+  structures, then exit.
+: All cache_dir types create the configured top-level directory
+  if it is missing. Other actions are type-specific.
+: This option does not enable validation of any present swap
+  structures. Its focus is on creation of missing pieces.
+: If nothing is missing, **squid -z** just exits.
+: If you suspect cache_dir corruption, you must delete the
+  top-level cache_dir directory before running **squid -z**.
+: By default, **squid -z** runs in daemon mode so that
+  configuration macros and other SMP features work as expected,
+  returning control to the caller before cache_dirs are fully
+  initialized.
+: If run from init scripts or daemon managers, the caller often
+  needs to wait for the initialization to complete before
+  proceeding further.
+: Use **"-\-foreground"** option to prevent premature exits.
+: Use **-N** option to disable daemon mode.
+
+# FILES
+
+Squid configuration files located in @SYSCONFDIR@/:
+
+squid.conf
+: The main configuration file.
+: You must initially make changes to this file for **squid** to
+  work.
```

Comment:
Removed obsolete mention of "http_access allow localnet" line being enabled by default.